src/fetch/payouts.py

```diff
@@ -295,7 +295,13 @@ def extend_payment_df(pdf: DataFrame, converter: TokenConversion) -> DataFrame:
     return pdf
 
 
-def prepare_transfers(payout_df: DataFrame, period: AccountingPeriod) -> PeriodPayouts:
+def prepare_transfers(
+    payout_df: DataFrame,
+    final_protocol_fee_wei: int,
+    integrators_fee_tax_wei: int,
+    integrators_fees_wei: dict[str, int],
```

Comment:
There are 3 types of transfers being created:
1) Reward (and fee) transfers to solvers
2) Protocol fee payment to the protocol fee safe
3) Partner fee payments
So potentially one could at least combine `integrators_fee_tax_wei` and `integrators_fee_wei` from 3) into the same `DataFrame`. Potentially we could even merge 2) and 3) as those are transfers which are not grouped by solver.